src/app/app.component.html

```diff
@@ -52,33 +56,33 @@
               <a routerLinkActive="active" href="javascript:void(0)" (click)="createWallet()">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-card-add" ></md-icon>
-                  <span class="menu-item">Add Acount</span>
+                  <span class="menu-item">Add Account</span>
                 </md-list-item>
               </a>
             </md-list>
           </md-content>
         </md-expansion-panel>
       </md-accordion>
     </md-content>
-
-    <!--md-content layout-padding>
+    
+    <!-- <md-content layout-padding>
       <md-accordion displayMode="flat" [multi]="'multi'" class="md-expansion-menu">
         <md-expansion-panel #panel1 expanded="true">
-          <md-expansion-panel-header>Settings</md-expansion-panel-header>
+          <md-expansion-panel-header *ngIf="isPinned">Settings</md-expansion-panel-header>
```

Comment:
I know it's commented, but let's take this ngIf out

Reply:
:)